azure-pipelines.yml

```diff
@@ -26,7 +26,7 @@ steps:
   displayName: 'Build for RISC-V'
 - script: |
     set -e
-    git clone https://github.com/qemu/qemu && cd qemu && git checkout 46517dd4971fc1fdd5b379e72cc377626ad98160
+    git clone https://github.com/qemu/qemu && cd qemu
```

Comment:
I'm being a bit stubborn here, but I'd still like to see a git checkout of the ref for which this patch is known to work. I don't want to get into this cycle we got into where a whole ton of PRs fail b/c our qemu patch is failing. Can you put the ref in for which this patch is known to work?

Reply:
I do see what you mean, but I don't like just hardcoding random SHAs. That leaves you open to relying on bugs in QEMU.

For the time being it's probably ok, once the next QEMU release comes out we can base it on stable releases which seems safe.